.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: python
 python:
+  # Python3.5 unit tests fail
```

Comment:
I created #36 to capture this work, can you refer to that in this comment, please?